client/deposits/utils/test/index.ts

```diff
@@ -6,7 +6,12 @@ import momentLib from 'moment';
 /**
  * Internal dependencies
  */
-import { getDepositDate, getDepositMonthlyAnchorLabel } from '../';
+import {
+	getDepositDate,
+	getDepositMonthlyAnchorLabel,
+	getNextDepositDate,
+} from '../';
+import * as AccountOverview from 'wcpay/types/account-overview';
```

Comment:
Nit: explicitly `import type` to make this easier to understand what's being imported.
```suggestion
import type * as AccountOverview from 'wcpay/types/account-overview';
```